### PR TITLE
Player controller using rigidbody and the option of kinematic is you want to for certain parts

### PR DESCRIPTION
**Pull request by AlecatEnjoyer**
updated player controller

### CODE REVIEW

**Review by marissalmoser — COMMENTED** [review]
My only comment is I think the looking up/down movement should be more sensitive to the mouse movement. We can ping a designer to get their opinion too

File: Assets/Scripts/Player/PlayerController.cs [hunk 1]
```diff
@@ -65,9 +74,9 @@ void Awake()
         move.performed += ctx => isMoving = true;
         move.canceled += ctx => moveDirection = move.ReadValue<Vector2>();
         move.canceled += ctx => isMoving = false;
-        //move.performed += ctx => rb.velocity = 
-        //    new Vector3(move.ReadValue<Vector2>().x * moveSpeed, rb.velocity.y, move.ReadValue<Vector2>().y * moveSpeed);
-        //move.canceled += ctx => rb.velocity = new Vector3(0, rb.velocity.y, 0);
+       // move.performed += ctx => rb.velocity =
```
Comment (alecpizz) [inline Assets/Scripts/Player/PlayerController.cs:77]: Can clean up this unecessary comments.

File: Assets/Scripts/Player/PlayerController.cs [hunk 2]
```diff
@@ -79,7 +88,6 @@ void Awake()
         //slash.canceled += ctx => cameraLens.FieldOfView = defaultFOV;
        // slash.canceled += ctx => StartZoom(false);
     }
-
     /// <summary>
```
Comment (alecpizz) [inline Assets/Scripts/Player/PlayerController.cs:91]: Clean up unused code.

File: Assets/Scripts/Player/PlayerController.cs [hunk 3]
```diff
@@ -141,17 +149,31 @@ void Awake()
     void FixedUpdate()
     {
         // Player Movement
-        if (isMoving)
+        if (isMoving == true && _isKinemat == true)
```
Comment (alecpizz) [inline Assets/Scripts/Player/PlayerController.cs:152]: This section is really verbose, things like == true are unecessary. You're also checking _isKinemat (should probably spell out the full word) every fixed update, when you could just do rb.isKinematic = _isKinemat.

File: Assets/Scripts/Player/PlayerController.cs [hunk 4]
```diff
@@ -166,7 +188,10 @@ void FixedUpdate()
 if (!isGrounded)
             isGrounded = Physics.CheckSphere(groundChecker.position, groundDistance, groundMask);
 
-
+        if (interact.IsPressed() && _doOnce == true)
```
Comment (alecpizz) [inline Assets/Scripts/Player/PlayerController.cs:191]: What's the purpose of this interaction system when we have existing one?

File: Assets/Scripts/Player/PlayerController.cs (hunk 3 above)
Comment (AlecatEnjoyer) [reply Assets/Scripts/Player/PlayerController.cs:152]: took out == parts except for false parts

File: Assets/Scripts/Player/PlayerController.cs (hunk 2 above)
Comment (AlecatEnjoyer) [reply Assets/Scripts/Player/PlayerController.cs:91]: sorry it was import code from another game from nick g I removed now

File: Assets/Scripts/Player/PlayerController.cs (hunk 1 above)
Comment (AlecatEnjoyer) [reply Assets/Scripts/Player/PlayerController.cs:77]: took out now

**Comment by AlecatEnjoyer** [comment]
> My only comment is I think the looking up/down movement should be more sensitive to the mouse movement. We can ping a designer to get their opinion too

made more sensitive 
-Mark

**Pull request merged**